specification/applicationinsights/resource-manager/microsoft.insights/stable/2015-05-01/analyticsItems_API.json

```diff
@@ -0,0 +1,401 @@
+                        "description": "Properties that need to be specified to create a new item and add it to an Application Insights component.",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                          "$ref": "#/definitions/ApplicationInsightsComponentAnalyticsItem"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The new or updated item associated with the Application Insights component.",
+                        "schema": {
+                            "$ref": "#/definitions/ApplicationInsightsComponentAnalyticsItem"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "AnalyticsItemPut": {
+                        "$ref": "./examples/AnalyticsItemPut.json"
+                    }
+                }
+            },
+            "delete": {
+                "description": "Deletes a specific Analytics Items defined within an Application Insights component.",
+                "operationId": "AnalyticsItem_Delete",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ItemScopePathParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ItemIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ItemNameParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The item has been successfully removed from the Application Insights component"
+                    }
+                },
+                "x-ms-examples": {
+                    "AnalyticsItemDelete": {
+                        "$ref": "./examples/AnalyticsItemDelete.json"
+                    }
+                }    
+            }
+        }
+    },
+    "definitions": {
+        "ApplicationInsightsComponentAnalyticsItem": {
+            "description": "Properties that define an Analytics item that is associated to an Application Insights component.",
+            "properties": {
+                "Id": {
+                    "type": "string",
+                    "description": "Internally assigned unique id of the item definition."
+                },
+                "Name": {
```

Comment:
since you're an existing service it's not unlikely but: Just wanna confirm, is PascalCasing really what your service sends over the wire?

Reply:
Unfortunately yes, this is how it was defined.